.github/workflows/build_test_windows_php7-4-21_vs15.yml

```diff
@@ -1,53 +0,0 @@
-# This is a basic workflow to help you get started with Actions
```

Comment:
Why was this file deleted?

Reply:
@sfc-gh-kwagner As I explained in the description, Windows 2016 was removed from github hosted runners since March 15 and VS 2017 gone with that as well. To build the connector for PHP7.4 VS2017 is required so we can't build that on github anymore unless we add self-hosted runner for VS 2017.